doc/INSTALL.md

```diff
@@ -235,6 +236,34 @@ Build with:
       make PIE=1 DEVELOPER=0 \
       CONFIGURATOR_CC="arm-linux-androideabi-clang -static"
 
+To cross-compile for Raspberry Pi
+--------------------
+
+Obtain the [official Raspberry Pi toolchains](https://github.com/raspberrypi/tools). This document assumes compilation will occur towards the Raspberry Pi 3 (arm-linux-gnueabihf as of Mar. 2018). In addition, obtain and install cross-compiled versions of sqlite 3 and gmp.
+
```

Comment:
Maybe better add sqlite3 and gmp cross build guide link that you mentioned in https://github.com/ElementsProject/lightning/issues/653#issuecomment-375424387

Reply:
Hi. I know this horrifically after-the-fact but I figured I should reply. :) Anyway, I think this is a good idea, although the PR has now been accepted. I'll think about whether or not I should make another one.